Containerfile.external-dns-operator

```diff
@@ -25,11 +25,14 @@ RUN git config --global --add safe.directory /workspace
 # Build
 RUN make build-operator
 
+#Define build arguments to recieve dynamic values
+ARG CI_VERSION
+
 FROM registry.access.redhat.com/ubi9/ubi:9.6-1751287003
 LABEL maintainer="Red Hat, Inc."
 LABEL com.redhat.component="external-dns-operator-container"
 LABEL name="external-dns-operator"
-LABEL version="1.3.0"
+LABEL version="${CI_VERSION}"
```

Comment:
I'm sorry I bugged on this one, it has to be `1.3.0` like after a run of `sync-version.sh`. https://github.com/openshift/external-dns-operator/pull/272#discussion_r2375544100.